sphinx/index.md

```diff
@@ -0,0 +1,43 @@
+# containerd
```

Comment:
For `docs/latest`, we should have this index.md and possibly `conf.py` in containerd/containerd.

For released versions such as 1.6, we decided to backport only security fixes. Do we want to backport documentation changes as well?

Reply:
> Is it okay to backport documentation changes from main to release/1.x?

Yes